libs/styles/styles.css

```diff
@@ -331,6 +331,15 @@
   text-decoration: none;
 }
 
+.m-button .con-button,
+.con-button.button-m {
+  border-radius: 21px;
+  font-size: 17px;
+  line-height: 21px;
+  min-height: 21px;
+  padding: 8px 22px 11px;
+}
```

Comment:
This new style looks almost exactly the same as `.button-l` and the 'medium' is intended to be the default .con-button. 
We shouldn't introduce this class here and instead have design settle with whats already established.

Reply:
Although it looks almost the same in code, it does look different visually compared side by side.

<img width="273" alt="Screenshot 2024-01-24 at 10 07 14 AM" src="https://github.com/adobecom/milo/assets/1179549/afffc0c4-bba2-46b9-9fbc-cc367454eacd">
<img width="221" alt="Screenshot 2024-01-24 at 10 07 07 AM" src="https://github.com/adobecom/milo/assets/1179549/ba64f5e7-27b0-4912-be76-e47402058bab">

Reply:
If you want to scope this minor padding differences for merch-card buttons, can you target them as button-l and add some type of override in your merch-card.css file? I don't want this added to styles.css. It's not consistently sized and looks visually like large. Who gave you these specks?

Reply:
Sure, I can do that. This [XD](https://xd.adobe.com/view/84200b4e-caf9-428f-98cf-7198a236a4bc-d173/screen/22b19a39-70bc-4b51-bd96-fd846bb8ef86/specs/) was provided by Curt Quentin from Consonant team.

Reply:
Essentially your saying you want a `.button-m` to look slightly bigger then the `.button-l`. That doesn't make much sense. :)

Reply:
it seems Milo has a wrong size for button L, according to consonant team. 

Heights for button as per Consonant:
S: 30px
M: 40px
L: 50px
 
<img width="641" alt="Screenshot 2024-01-24 at 10 30 55 AM" src="https://github.com/adobecom/milo/assets/1179549/53058f2c-68c7-40b1-84f7-26709534ff73">

<img width="280" alt="Screenshot 2024-01-24 at 10 29 47 AM" src="https://github.com/adobecom/milo/assets/1179549/340ad83e-bd3e-48a4-99a1-89a5ab7ba7bc">

Reply:
For now, I'll use button L since that is the height we need for this CTAs. But something to keep in mind regarding the token discrepancy between XD and Milo.

Reply:
Thanks for the call out on that and splitting hairs, there is indeed a discrepancy on those labels. Something to keep it in mind.
Maybe we can add button size documentation and call that out.